### DISCUSSION TITLE
Wrap output

### DISCUSSION THREAD

**Discussion by UVLabs** [spatie/ray#784] [Q&A]
How can an output be wrapped so that it appears on the second line? I'm outputting some base64 and I can't copy everything because it's trimmed..

**Comment by freekmurze** [comment]
Could you try with `ray()->text($variable)`?

**Reply by UVLabs** [reply]
That didn't help, you can see this video: https://share.vidyard.com/watch/b7aMXx4oFJT6QZTQ3ZLJrf?

It shows me expanding the ray window to see the rest of the text that exists outside the ray viewport

**Comment by justrustyspoons** [comment]
I have the same issue, long strings just go off the edge of the screen. Needs something like CSS 'overflow-wrap: break-word'

**Comment by UVLabs** [comment]
This is still an issue...I tried to output some base64 and its pointless because ray doesn't show all the content
